rebar.config

```diff
@@ -61,12 +59,12 @@
                 opentelemetry_zipkin_pb]}.
 
 
-{dialyzer, [{warnings, [no_unknown]}]}.
+%{dialyzer, [{warnings, [no_unknown]}]}.
```

Comment:
Q: is it a todo ?

Reply:
Not sure, but this conf causes dialyzer errors:
```
===> Error in dialyzing apps: Unknown dialyzer warning option: no_unknown
```
I commented it out and dialyzer worked fine...